open-apparel-registry/open-apparel-registry
# Change production type to Processing type in Django admin

## Overview

Change production type filed on the FacilityClaim model in Django admin to processing type to match the name change on the facility claim front-end.

Connects #1669 

## Demo

<img width="507" alt="image" src="https://user-images.githubusercontent.com/60887686/155741185-e4334747-fa8d-48fa-8f36-7c521bc3ebe1.png">

## Testing Instructions

* Checkout this branch, go to the [admin](http://localhost:8081/admin/api/facilityclaim/add/) site to create a new claim and log in as c1@example.com
* "Processing type" should be below the "Production type"
* Try to create a new claim and things should be working as expected.

## Checklist
